project-base/src/Shopsys/ShopBundle/Component/Flysystem/VolumeDriver.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
You are declaring strict_types, but you are not using type hints

Reply:
`GetUrl::getFromMethod(string $path): string should be compatible with Barryvdh\elFinderFlysystemDriver\Plugin\GetUrl::getFromMethod($path)`
we cant redeclare extended methods so i will remove strict_type declarations then.